proto/dydxprotocol/indexer/events/events.proto

```diff
@@ -282,9 +282,10 @@ message StatefulOrderEventV1 {
     dydxprotocol.indexer.protocol.v1.IndexerOrder order = 1;
   }
 
-  // A long term order placement contains an order.
+  // A long term order replacement contains an old order ID and the new order.
   message LongTermOrderReplacementV1 {
-    dydxprotocol.indexer.protocol.v1.IndexerOrder order = 1;
+    dydxprotocol.indexer.protocol.v1.IndexerOrderId old_order_id = 1;
```

Comment:
Can you add a comment about why this is needed? Vaults replace orders with different order id